### PR TITLE
Initial Presentation

### PR DESCRIPTION
**Pull request by qubies**
Feel free to offer comments or improvements prior to the presentation.

### CODE REVIEW

File: 2017-11/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Concurrency in Go: an introduction to channels.
+
+Channels are one of the most interesting features in GoLang. This talk shows some of the basics of working with concurrent applications, go routines and channels. This repository contains the demonstration code. The code is ocasionally intentionally broken to highlight different aspects of concurrency mechanisms.
```
Comment (nathany) [inline 2017-11/README.md]: Go~~Lang~~ Just Go. `#golang` is just used for a tag and the golang.org website (Disney owns go.com)

ocasionally -> occasionally (spell checkers are handy)

Comment (qubies) [reply 2017-11/README.md]: Thanks for that! I use vim for my editor, and when coding all the red of the spell checker gets distracting, so its usually off.

**Comment by nathany** [comment]
@qubies If it's easier to keep all your presentation files together at https://github.com/qubies/ConcurrencyLecture, that's okay too. Either way we'll link it up from the website.

**Comment by qubies** [comment]
It was easier for me to include it as a submodule at the time, it doesn't matter to me now :) Do you want me to include the files directly here?

**Comment by nathany** [comment]
Including the files here would be nice (instead of the submodule).

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "2017-11/ConcurrencyLecture"]
```
Comment (nathany) [inline .gitmodules]: please remove the submodule if it's no longer needed

Comment (qubies) [reply .gitmodules]: apologies again! Removed.

**Comment by nathany** [comment]
Hey Tobias. Unless you have any further tweaks, feel free to hit merge on this!

Thanks again for your presentation. Appreciated.

**Pull request merged**